src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/DocAssemblyService.java

```diff
@@ -68,12 +68,31 @@ public DocAssemblyResponse createOrder(CCDCase ccdCase, String authorisation) {
         );
     }
 
+    public DocAssemblyResponse createGeneralLetter(CCDCase ccdCase, String authorisation, String templateId) {
+        UserDetails userDetails = userService.getUserDetails(authorisation);
+
+        logger.info("Doc assembly service: creating request for doc assembly");
```

Comment:
Logging could be more descriptive
```suggestion
        logger.info("Doc assembly service: creating general letter request for doc assembly");
```